logstash-event.gemspec

```diff
@@ -25,4 +26,11 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rspec"
   gem.add_development_dependency "guard"
   gem.add_development_dependency "guard-rspec"
```

Comment:
The guard* gems can be safely removed, we don't have a **Guardfile** in the project.